workers/jobs/google_scraper.go

```diff
@@ -33,7 +32,7 @@ func (c *Context) PerformSearchJob(job *work.Job) error {
 	keywordID := uint(job.ArgInt64("keywordID"))
 	keyword := job.ArgString("keyword")
 	if keywordID == 0 {
-		log.Printf("Cannot perform job (reason: %v)", invalidKeywordIDError)
+		log.Error("Failed to perform job: ", invalidKeywordIDError)
```

Comment:
@gutakk Should we add this as a constant in `helpers/error_handler/error_constant.go`?

Reply:
Same as https://github.com/gutakk/go-google-scraper/pull/78#discussion_r573613521